src/io/fabric8/Fabric8Commands.groovy

```diff
@@ -453,7 +453,7 @@ def getIssueComments(project, id, githubToken = null) {
     def apiUrl = new URL("https://api.github.com/repos/${project}/issues/${id}/comments")
     echo "getting comments for ${apiUrl}"
 
-    def HttpURLConnection connection = apiUrl.openConnection()
+    HttpURLConnection connection = apiUrl.openConnection()
```

Comment:
new to groovy ... could you please point to the rule it breaks?

Reply:
@sthaha def is kind of untyped var definition, and if you define a variable with an explicit type, its not necessary. http://www.groovy-lang.org/semantics.html#_variable_definition